From 7698080e20b03bec76fed175d90fc560ce77d514 Mon Sep 17 00:00:00 2001
From: WenboZhu <WenboZhu@freescale.com>
Date: Wed, 29 Apr 2015 14:21:50 +0800
Subject: [PATCH 094/115] MA-6612 the sabresd 7d can't support A2dpSink(UI is
 too large for sabresd 7d)

add a new file layout-480x224 and mv layout to layout-1024x720

Signed-off-by: WenboZhu <WenboZhu@freescale.com>
---
 A2dpSinkApp/res/layout-1024x720/activity_main.xml | 108 ++++++++++++++++++++++
 A2dpSinkApp/res/layout-480x224/activity_main.xml  | 108 ++++++++++++++++++++++
 A2dpSinkApp/res/layout/activity_main.xml          | 108 ----------------------
 3 files changed, 216 insertions(+), 108 deletions(-)
 create mode 100755 A2dpSinkApp/res/layout-1024x720/activity_main.xml
 create mode 100755 A2dpSinkApp/res/layout-480x224/activity_main.xml
 delete mode 100755 A2dpSinkApp/res/layout/activity_main.xml

diff --git a/A2dpSinkApp/res/layout-1024x720/activity_main.xml b/A2dpSinkApp/res/layout-1024x720/activity_main.xml
new file mode 100755
index 0000000..dd4ca95
--- /dev/null
+++ b/A2dpSinkApp/res/layout-1024x720/activity_main.xml
@@ -0,0 +1,108 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin" tools:context=".MainActivity">
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:gravity="center">
+    <LinearLayout
+        android:layout_width="fill_parent"
+        android:layout_height="250dp"
+        android:gravity="center"
+        android:id="@+id/linearLayout">
+        <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="250dp"
+        android:layout_height="250dp"
+        android:gravity="center"
+        android:weightSum="1">
+
+
+            <ImageView
+                android:layout_width="200dp"
+                android:id="@+id/img_a2dpState"
+                android:layout_height="200dp"
+                android:background="@drawable/a2dp_no_connected"/>
+            <TextView
+                android:layout_width="200dp"
+                android:layout_height="30dp"
+                android:text="@string/noConnected"
+                android:id="@+id/txt_a2dpState"
+                android:textStyle="bold"
+                android:gravity="top|center" />
+
+        </LinearLayout>
+    </LinearLayout>
+
+    <LinearLayout
+        android:orientation="horizontal"
+        android:layout_width="fill_parent"
+        android:layout_height="120dp"
+        android:layout_alignParentStart="true"
+        android:layout_below="@+id/linearLayout"
+        android:gravity="center">
+
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="90dp"
+            android:layout_height="90dp"
+            android:id="@+id/btn_previous"
+            android:background="@drawable/previous"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="90dp"
+            android:layout_height="90dp"
+            android:id="@+id/btn_playi"
+            android:background="@drawable/play"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="90dp"
+            android:layout_height="90dp"
+            android:id="@+id/btn_pausei"
+            android:background="@drawable/pause"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="90dp"
+            android:layout_height="90dp"
+            android:id="@+id/btn_next"
+            android:background="@drawable/next"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        </LinearLayout>
+        <LinearLayout
+            android:layout_width="fill_parent"
+            android:layout_height="100dp"
+            android:gravity="center">
+            <Button
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/start"
+                android:id="@+id/btn_start"
+                android:visibility="invisible"
+                />
+            <ImageButton
+                android:layout_marginLeft="15dp"
+                android:layout_width="110dp"
+                android:layout_height="110dp"
+                android:id="@+id/btn_enable"
+                android:background="@drawable/power_off"/>
+            <Button
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/stop"
+                android:id="@+id/btn_stop"
+                android:visibility="invisible"/>
+            </LinearLayout>
+    </LinearLayout>
+</RelativeLayout>
diff --git a/A2dpSinkApp/res/layout-480x224/activity_main.xml b/A2dpSinkApp/res/layout-480x224/activity_main.xml
new file mode 100755
index 0000000..9159a84
--- /dev/null
+++ b/A2dpSinkApp/res/layout-480x224/activity_main.xml
@@ -0,0 +1,108 @@
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
+    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin" tools:context=".MainActivity">
+
+    <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="fill_parent"
+        android:layout_height="fill_parent"
+        android:gravity="center">
+    <LinearLayout
+        android:layout_width="fill_parent"
+        android:layout_height="100dp"
+        android:gravity="center"
+        android:id="@+id/linearLayout">
+        <LinearLayout
+        android:orientation="vertical"
+        android:layout_width="250dp"
+        android:layout_height="125dp"
+        android:gravity="center"
+        android:weightSum="1">
+
+
+            <ImageView
+                android:layout_width="50dp"
+                android:id="@+id/img_a2dpState"
+                android:layout_height="50dp"
+                android:background="@drawable/a2dp_no_connected"/>
+            <TextView
+                android:layout_width="600dp"
+                android:layout_height="25dp"
+                android:text="@string/noConnected"
+                android:id="@+id/txt_a2dpState"
+                android:textStyle="bold"
+                android:gravity="top|center" />
+
+        </LinearLayout>
+    </LinearLayout>
+
+    <LinearLayout
+        android:orientation="horizontal"
+        android:layout_width="fill_parent"
+        android:layout_height="30dp"
+        android:layout_alignParentStart="true"
+        android:layout_below="@+id/linearLayout"
+        android:gravity="center">
+
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="30dp"
+            android:layout_height="30dp"
+            android:id="@+id/btn_previous"
+            android:background="@drawable/previous"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="30dp"
+            android:layout_height="30dp"
+            android:id="@+id/btn_playi"
+            android:background="@drawable/play"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="30dp"
+            android:layout_height="30dp"
+            android:id="@+id/btn_pausei"
+            android:background="@drawable/pause"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        <ImageButton
+            android:layout_marginLeft="10dp"
+            android:layout_width="30dp"
+            android:layout_height="30dp"
+            android:id="@+id/btn_next"
+            android:background="@drawable/next"
+            android:layout_centerVertical="true"
+            android:layout_toStartOf="@+id/btn_play" />
+        </LinearLayout>
+        <LinearLayout
+            android:layout_width="fill_parent"
+            android:layout_height="40dp"
+            android:gravity="center">
+            <Button
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/start"
+                android:id="@+id/btn_start"
+                android:visibility="invisible"
+                />
+            <ImageButton
+                android:layout_marginLeft="15dp"
+                android:layout_width="40dp"
+                android:layout_height="40dp"
+                android:id="@+id/btn_enable"
+                android:background="@drawable/power_off"/>
+            <Button
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/stop"
+                android:id="@+id/btn_stop"
+                android:visibility="invisible"/>
+            </LinearLayout>
+    </LinearLayout>
+</RelativeLayout>
diff --git a/A2dpSinkApp/res/layout/activity_main.xml b/A2dpSinkApp/res/layout/activity_main.xml
deleted file mode 100755
index dd4ca95..0000000
--- a/A2dpSinkApp/res/layout/activity_main.xml
+++ /dev/null
@@ -1,108 +0,0 @@
-<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools" android:layout_width="match_parent"
-    android:layout_height="match_parent" android:paddingLeft="@dimen/activity_horizontal_margin"
-    android:paddingRight="@dimen/activity_horizontal_margin"
-    android:paddingTop="@dimen/activity_vertical_margin"
-    android:paddingBottom="@dimen/activity_vertical_margin" tools:context=".MainActivity">
-
-    <LinearLayout
-        android:orientation="vertical"
-        android:layout_width="fill_parent"
-        android:layout_height="fill_parent"
-        android:gravity="center">
-    <LinearLayout
-        android:layout_width="fill_parent"
-        android:layout_height="250dp"
-        android:gravity="center"
-        android:id="@+id/linearLayout">
-        <LinearLayout
-        android:orientation="vertical"
-        android:layout_width="250dp"
-        android:layout_height="250dp"
-        android:gravity="center"
-        android:weightSum="1">
-
-
-            <ImageView
-                android:layout_width="200dp"
-                android:id="@+id/img_a2dpState"
-                android:layout_height="200dp"
-                android:background="@drawable/a2dp_no_connected"/>
-            <TextView
-                android:layout_width="200dp"
-                android:layout_height="30dp"
-                android:text="@string/noConnected"
-                android:id="@+id/txt_a2dpState"
-                android:textStyle="bold"
-                android:gravity="top|center" />
-
-        </LinearLayout>
-    </LinearLayout>
-
-    <LinearLayout
-        android:orientation="horizontal"
-        android:layout_width="fill_parent"
-        android:layout_height="120dp"
-        android:layout_alignParentStart="true"
-        android:layout_below="@+id/linearLayout"
-        android:gravity="center">
-
-        <ImageButton
-            android:layout_marginLeft="10dp"
-            android:layout_width="90dp"
-            android:layout_height="90dp"
-            android:id="@+id/btn_previous"
-            android:background="@drawable/previous"
-            android:layout_centerVertical="true"
-            android:layout_toStartOf="@+id/btn_play" />
-        <ImageButton
-            android:layout_marginLeft="10dp"
-            android:layout_width="90dp"
-            android:layout_height="90dp"
-            android:id="@+id/btn_playi"
-            android:background="@drawable/play"
-            android:layout_centerVertical="true"
-            android:layout_toStartOf="@+id/btn_play" />
-        <ImageButton
-            android:layout_marginLeft="10dp"
-            android:layout_width="90dp"
-            android:layout_height="90dp"
-            android:id="@+id/btn_pausei"
-            android:background="@drawable/pause"
-            android:layout_centerVertical="true"
-            android:layout_toStartOf="@+id/btn_play" />
-        <ImageButton
-            android:layout_marginLeft="10dp"
-            android:layout_width="90dp"
-            android:layout_height="90dp"
-            android:id="@+id/btn_next"
-            android:background="@drawable/next"
-            android:layout_centerVertical="true"
-            android:layout_toStartOf="@+id/btn_play" />
-        </LinearLayout>
-        <LinearLayout
-            android:layout_width="fill_parent"
-            android:layout_height="100dp"
-            android:gravity="center">
-            <Button
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/start"
-                android:id="@+id/btn_start"
-                android:visibility="invisible"
-                />
-            <ImageButton
-                android:layout_marginLeft="15dp"
-                android:layout_width="110dp"
-                android:layout_height="110dp"
-                android:id="@+id/btn_enable"
-                android:background="@drawable/power_off"/>
-            <Button
-                android:layout_width="wrap_content"
-                android:layout_height="wrap_content"
-                android:text="@string/stop"
-                android:id="@+id/btn_stop"
-                android:visibility="invisible"/>
-            </LinearLayout>
-    </LinearLayout>
-</RelativeLayout>
-- 
1.8.0

